src/nls/messages_en.json

```diff
@@ -97,14 +92,15 @@
     "dashboard.notification.clear": "Clear Notification",
     "dashboard.paramaters.title": "Parameters",
     "dashboard.parameters.title": "Parameters",
-    "dashboard.pipelineResource.createPipelineResourceTitle": "Create PipelineResource",
     "dashboard.pipelineResource.errorLoading": "Error loading PipelineResource",
     "dashboard.pipelineResource.failed": "Cannot load PipelineResource",
     "dashboard.pipelineResource.failedSubtitle": "PipelineResource {pipelineResourceName} not found",
     "dashboard.pipelineResource.fieldName": "Field Name",
     "dashboard.pipelineResource.secretKey": "Secret Key",
     "dashboard.pipelineResource.secretName": "Secret Name",
     "dashboard.pipelineResources.createSuccess": "Successfully created PipelineResource",
+    "dashboard.pipelineResources.deleteConfirm": "Are you sure you want to delete these PipelineResources?",
```

Comment:
Are the pipeline resources listed in the message ?

Reply:
Yes similar to the experience for Secrets, we display an unordered list of the PipelineResource names below this prompt.